java/configuration/src/main/java/sleeper/configuration/properties/table/TableProperty.java

```diff
@@ -176,9 +176,9 @@ public interface TableProperty extends SleeperProperty {
             .defaultProperty(DEFAULT_COMPACTION_FILES_BATCH_SIZE)
             .description("The number of files to read in a compaction job. Note that the state store " +
                     "must support atomic updates for this many files.\n" +
-                    "The DynamoDBStateStore must be able to atomically apply 2 updates to create the output files for a " +
-                    "splitting compaction, and 2 updates for each input file to mark them as ready for garbage " +
-                    "collection. There's a limit of 100 atomic updates, which equates to 48 files in a compaction.\n" +
+                    "The DynamoDBStateStore must be able to atomically apply 2 updates for each input file to mark " +
+                    "them as ready for garbage collection. There's a limit of 100 atomic updates, which equates to " +
+                    "48 files in a compaction.\n" +
```

Comment:
See above comment and 48 should be 49 now. And I think we can update the description as we now remove file references rather than marking them as ready for garbage collection.

Reply:
Done, thanks